app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError

```

Comment:
format should be like this from app.errors import (NotVaccinatedError,
                                                                                     NotWearingMaskError) 
check readme_file check_list!